skimage/feature/blob.py

```diff
@@ -31,8 +31,8 @@ def _compute_disk_overlap(d, r1, r2):
 
     Returns
     -------
-    vol: float
-        Volume of the overlap between the two disks.
+    area: float
```

Comment:
Should it be `fraction: float` then? Or just `ret: float`.
Same for `_compute_sphere_overlap`.